src/codeu/chat/server/Server.java

```diff
@@ -83,6 +83,15 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    /*The purpose of this command is to pass the 'startTime' to the
+    client, when requested by him/her. */
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+        Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+        Time.SERIALIZER.write(out, view.getInfo().startTime);
```

Comment:
move line 92 to line 83 since we have the same key already in the map it will override the previous one